scipy/sparse/csgraph/_flow.pyx

```diff
@@ -496,15 +496,16 @@ cdef bint _build_level_graph(
 
 cdef ITYPE_t _augment_paths(
         const ITYPE_t[:] edge_ptr,  # IN
-        const ITYPE_t current,  # IN
+        const ITYPE_t source,  # IN
         const ITYPE_t sink,  # IN
         const ITYPE_t[:] levels,  # IN
         const ITYPE_t[:] heads,  # IN
         const ITYPE_t[:] rev_edge_ptr,  # IN
         ITYPE_t[:] capacities,  # IN/OUT
         ITYPE_t[:] progress,  # IN
         ITYPE_t[:] flows,  # OUT
-        ITYPE_t flow  # OUT
+        ITYPE_t flow,  # OUT
```

Comment:
Since we're recursive now, there's limited need to pass in a `flow` (which will always be `ITYPE_MAX`).